.github/workflows/daily_ete_test.yml

```diff
@@ -32,7 +32,7 @@ on:
         required: true
         description: 'Dependency packages, you can also set a specific version'
         type: string
-        default: 'packaging transformers_stream_generator transformers datasets matplotlib openai attrdict timm modelscope jmespath'
+        default: 'packaging transformers_stream_generator transformers==4.41.2 datasets matplotlib openai attrdict timm modelscope jmespath'
```

Comment:
Can we upgrade transformers to `4.42.3`?

Reply:
> Can we upgrade transformers to `4.42.3`?

4bits quantization of vl model is still fail on 4.42.3 version

Reply:
@AllentDan may follow this up